modules/edu.gemini.seqexec.engine/src/test/scala/edu/gemini/seqexec/engine/packageSpec.scala

```diff
@@ -188,7 +188,10 @@ class packageSpec extends FlatSpec with NonImplicitAssertions {
   }
 
   "action state" should "change to Paused if output is Paused" in {
-    assert(actionPause.map(_.sequences(seqId).current.execution.forall(_.state.runState === Action.Paused)).getOrElse(false))
+    assert(actionPause.map(_.sequences(seqId).current.execution.forall{_.state.runState match {
```

Comment:
the match for `Action.Paused` seems a common idiom. Perhaps it could be a method to do `runState.isPaused`?